problem1.py

```diff
@@ -1 +1,19 @@
+'''
+-------------------------------------------------------------------------------
+Name:		problem1.py
+Purpose:	Converting celsius to fahrenheit
 
+Author:	Huang.S
+
+Created:	date in 07/12/2020
+------------------------------------------------------------------------------
+'''
+
+# Input celsius 
+celsius = float(input("Please input the temperature in celsius: "))
+
+# Calculations
+fahrenheit = (9 /5 *celsius +  32)
```

Comment:
whitespacing could be better here `fahrenheit = (9 /5 *celsius +  32)` --> `fahrenheit = (9/5*celsius + 32)`